styleguide/source/_patterns/01-molecules/06-components/11-hub-card/hub-card-portrait.twig

```diff
@@ -0,0 +1,8 @@
+{#
+ # @file hub-card-portrait.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<a class="hub-card hub-card-portrait hub-card-image-123" href="#">
```

Comment:
Since the image is the primary content of this card, does it still make sense to display it as a background image? That may have accessibility concerns. Also, I'm not sure we want to crop a portrait image as aggressively... What would the implications be of not making this a background image?